src/server.js

```diff
@@ -25,9 +23,8 @@ app.get('/', (req, res) => res.json({ msg: 'Welcome to Sylica REST API' }));
 // define router paths
 app.use('/api/auth', authRouter);
 app.use('/api/products', productsRouter);
-app.use('/api/checkout', checkoutRouter)
-
-
+app.use('/api/checkout', checkoutRouter);
+app.use('/api/orders',authenticateToken,orderRouter);
```

Comment:
authenticateToken goes here `router.post('/create-order', createOrder)`, not where you use the route